_src/_assets/js/dashboard-map.js

```diff
@@ -117,15 +183,17 @@
       .attr('id', 'map-tooltip')
       .style('display', 'none')
 
-    const maxValue = d3.max(joinedData.features, d => getValue(d))
+    const maxValue = d3.max(joinedData.features, d =>
+      getValue(d, 'totalTestResults'),
+    )
 
     const r = d3
       .scaleSqrt()
       .domain([0, maxValue])
       .range([0, 50])
     const map = svg.append('g')
-    const bubbles = svg.append('g')
-    const deathBubbles = svg.append('g')
+    const testBubbles = svg.append('g').attr('id', 'testBubbles')
+    const bubbles = svg.append('g').attr('id', 'bubbles')
```

Comment:
What is the difference between `testBubbles` and `bubbles`?

Reply:
the map now has concentric circles, one for test #s and one for positives.  I was having trouble using a single element to contain them so i just ended up using two.